crates/bevy_winit/src/lib.rs

```diff
@@ -316,19 +199,21 @@ impl Default for WinitPersistentState {
     }
 }
 
-#[derive(Default, Resource)]
-struct WinitCreateWindowReader(ManualEventReader<CreateWindow>);
+// #[derive(Default)]
+// struct WinitCreateWindowReader(ManualEventReader<CreateWindow>);
```

Comment:
Remove if no longer useful

Reply:
Ya I think I accidentally re-added this when rebasing